_posts/2017-10-18-understanding-colourblindness.md

```diff
@@ -1,43 +1,37 @@
-description: "What is colour blindness, and how can we design with it in mind?"
+title: "Understanding colorblindness"
+description: "What is color blindness, and how can we design with it in mind?"
 author: matthew_higgins
 published: true
 categories:
   - Background
 ---
 
-Some peoples' eyes can't tell the difference between certain colours; that's not to say they see everything in greyscale. Total colour blindness is very rare in humans.
+Colorblindness is a common genetic condition in which some peoples' eyes can't tell the difference between certain colors. That's not to say they see everything in greyscale-- just that many colors seem identical to one another. Total color blindness is very rare in humans.
 
-Yet, colour vision deficiencies (typically grouped together as "colour blindness") are not rare. Some form of colour blindness affects at least one in twenty people; [Wikipedia has a helpful table](https://en.wikipedia.org/wiki/Color_blindness#Epidemiology), based on [a journal article](https://apfmj.biomedcentral.com/articles/10.1186/s12930-014-0010-3). There are [several different forms](http://www.colourblindawareness.org/colour-blindness/types-of-colour-blindness/). The most common, red-green colour blindness, affects around one in twelve men.
+Color vision deficiencies (typically grouped together as "color blindness") are not rare. Some form of color blindness affects at least one in twenty people. [Wikipedia has a helpful table](https://en.wikipedia.org/wiki/Color_blindness#Epidemiology), based on [a journal article](https://apfmj.biomedcentral.com/articles/10.1186/s12930-014-0010-3). There are [several different forms](http://www.colourblindawareness.org/colour-blindness/types-of-colour-blindness/). The most common, red-green color blindness, affects around one in twelve men.
 
-Colour blindness is less profound than many impairments. Yet, people with colour vision problems often face restrictions on certain activities. This varies around the world. For example, Romania prohibits residents who are colour blind from driving. Yet, the USA doesn't consider colour blindness as a disability, for workplace discrimination purposes.
+Color blindness is less profound than many impairments, but people with color vision problems often face restrictions on certain activities. This varies around the world. For example, Romania prohibits residents who are color blind from driving. The USA doesn't consider color blindness a disability because of workplace discrimination purposes.
 
-I've avoided going into the different types here, as the principles are the same for each. Also, [Wikipedia's article on colour blindness](https://en.wikipedia.org/wiki/Color_blindness) has some excellent descriptions.
+Though there are multiple types of color blindness, they are the same in principle. For more information, check out [Wikipedia's article on colour blindness](https://en.wikipedia.org/wiki/Color_blindness), which has some excellent descriptions.
```

Comment:
to match other instances:
Wikipedia's article on **colour** blindness => Wikipedia's article on **color** blindness

Reply:
@scottaohara @ericwbailey Thank you for catching those!